api/hooks.js

```diff
@@ -111,6 +115,8 @@ function proxyGlobalEvents (global, target) {
         dispatchEvent(target, new MessageEvent(type, { origin, data, detail }))
       } else if (detail) {
         dispatchEvent(target, new CustomEvent(type, { detail }))
+      } else if (detail) {
```

Comment:
this branch will never execute

Reply:
It was supposed to be URL, but I think I Ctl-z'd it back to detail.